src/lib/staged_ledger/staged_ledger.ml

```diff
@@ -520,11 +520,12 @@ module T = struct
       (pending_coinbase_stack_state : Stack_state_with_init_stack.t) s
       txn_state_view =
     let open Result.Let_syntax in
+    (*TODO: check fee_excess as a result of applying the txns matches with this*)
```

Comment:
As noted above: `Ledger.apply_transaction` should return this, and this call should disappear.